files/install_config_agent_yum.sh

```diff
@@ -11,6 +11,9 @@ set -eux
 exec > >(tee -a /var/log/bash_script.log) 2>&1
 set -x
 
+#connect eth1 and use dhcp to get address from neutron network
```

Comment:
Not sure this here is a good idea. The file naming is confusing and was orignally there to setup os-collect-*. Can we split out the OpenShift prep stuff into a separate file?